src/core/user-wallet.ts

```diff
@@ -35,13 +35,14 @@ export abstract class UserWallet {
   }
 
   async getBalances(lock?): Promise<Balances> {
-    Wallets.updatePendingInvoices({
-      walletId: this.user.id as WalletId,
+    // was the await omit on purpose?
```

Comment:
Yes, they both were originally left without the `await` and then we added it back to the `Wallets.updatePendingPayments` line to be able to see if there were any errors to return from. We didn't need the error feedback from `Wallets.updatePendingInvoices` so we left it without the await, though I don't remember why we made this distinction at the time.

Reply:
I feel a comment would be nice to have in future PR. will leave the comment for now on if that's ok.